src/web/components/SmallHeadline.tsx

```diff
@@ -119,6 +127,7 @@ export const SmallHeadline = ({
                     underlined && underlinedStyles(size),
                     showUnderline && textDecorationUnderline,
                     coloured && colourStyles(palette[pillar].dark),
+                    linkTo && visitedColour && visitedStyles(visitedColour),
```

Comment:
Small thing but I guess we could allow the `visitedColour` to be set even without a `linkTo`? It's perhaps not logical but there's no hard dependency on having the `linkTo` prop before setting this style.

Reply:
The way I see it, there's no hard dependency on `linkTo' to set this `visitedColour` as you say, but I thought it wouldn't make sense to actually implement `visitedColour` if we don't have a link. So I'm stopping `visitedColour` from having any effect whatsoever as early as possible in the component lifecycle.